examples/tpm2_filternv/README.md

```diff
@@ -0,0 +1,17 @@
+# tpm2_nvfilter
+
+Display a list of NV indices which matches zero or more attributes, such as `written` or `authread`.
+```
```

Comment:
You could use ` ```console` to format this snippet as an interactive shell session.

Reply:
Ah, missed that console exists, it's added now